tests/unit_tests/test_wave_02.py

```diff
@@ -34,7 +34,7 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("electronics")
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
```

Comment:
❗ Besides commenting out this Exception, you need to write assertions for this test too